ethcore/src/client/evm_test_client.rs

```diff
@@ -87,7 +88,9 @@ impl<'a> EvmTestClient<'a> {
 			ForkSpec::Homestead => Some(&*HOMESTEAD),
 			ForkSpec::EIP150 => Some(&*EIP150),
 			ForkSpec::EIP158 => Some(&*EIP161),
-			ForkSpec::Metropolis | ForkSpec::Byzantium | ForkSpec::Constantinople => None,
+			ForkSpec::Byzantium => Some(&*BYZANTIUM),
+			ForkSpec::EIP158ToByzantiumAt5 => Some(&BYZANTIUM_TRANSITION),
+			_ => None,
```

Comment:
I preferred explicit enumeration here to get a compile error if you add a new one in `ForkSpec`.